From a1b261b559e4945ea87a81757b863b7074ea9819 Mon Sep 17 00:00:00 2001
From: Clement Faure <clement.faure@nxp.com>
Date: Tue, 17 Apr 2018 14:43:21 +0200
Subject: [PATCH 3655/5242] MLK-18036-2 Delete *optee.dts files

commit  ad035b4011156a663fb5c846ad395e87f720582e from
https://source.codeaurora.org/external/imx/linux-imx.git

A specific node for OCRAM mapping in optee as been added in
the device tree. These dedicated optee device trees can be
removed.

Signed-off-by: Clement Faure <clement.faure@nxp.com>
Acked-by: Peng Fan <peng.fan@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm/boot/dts/Makefile                    |   14 --------------
 arch/arm/boot/dts/imx6dl-sabreauto-optee.dts  |   13 -------------
 arch/arm/boot/dts/imx6dl-sabresd-optee.dts    |   13 -------------
 arch/arm/boot/dts/imx6q-sabreauto-optee.dts   |   14 --------------
 arch/arm/boot/dts/imx6q-sabresd-optee.dts     |   14 --------------
 arch/arm/boot/dts/imx6qp-sabreauto-optee.dts  |   14 --------------
 arch/arm/boot/dts/imx6qp-sabresd-optee.dts    |   14 --------------
 arch/arm/boot/dts/imx6sl-evk-optee.dts        |   13 -------------
 arch/arm/boot/dts/imx6sll-evk-optee.dts       |   13 -------------
 arch/arm/boot/dts/imx6sx-sabreauto-optee.dts  |   23 -----------------------
 arch/arm/boot/dts/imx6sx-sdb-optee.dts        |   23 -----------------------
 arch/arm/boot/dts/imx6ul-14x14-evk-optee.dts  |   14 --------------
 arch/arm/boot/dts/imx6ul-9x9-evk-optee.dts    |   13 -------------
 arch/arm/boot/dts/imx6ull-14x14-evk-optee.dts |   13 -------------
 arch/arm/boot/dts/imx7d-sdb-optee.dts         |   22 ----------------------
 15 files changed, 230 deletions(-)
 delete mode 100644 arch/arm/boot/dts/imx6dl-sabreauto-optee.dts
 delete mode 100644 arch/arm/boot/dts/imx6dl-sabresd-optee.dts
 delete mode 100644 arch/arm/boot/dts/imx6q-sabreauto-optee.dts
 delete mode 100644 arch/arm/boot/dts/imx6q-sabresd-optee.dts
 delete mode 100644 arch/arm/boot/dts/imx6qp-sabreauto-optee.dts
 delete mode 100644 arch/arm/boot/dts/imx6qp-sabresd-optee.dts
 delete mode 100644 arch/arm/boot/dts/imx6sl-evk-optee.dts
 delete mode 100644 arch/arm/boot/dts/imx6sll-evk-optee.dts
 delete mode 100644 arch/arm/boot/dts/imx6sx-sabreauto-optee.dts
 delete mode 100644 arch/arm/boot/dts/imx6sx-sdb-optee.dts
 delete mode 100644 arch/arm/boot/dts/imx6ul-14x14-evk-optee.dts
 delete mode 100644 arch/arm/boot/dts/imx6ul-9x9-evk-optee.dts
 delete mode 100644 arch/arm/boot/dts/imx6ull-14x14-evk-optee.dts
 delete mode 100644 arch/arm/boot/dts/imx7d-sdb-optee.dts

diff --git a/arch/arm/boot/dts/Makefile b/arch/arm/boot/dts/Makefile
index 443969d..2f8197b 100644
--- a/arch/arm/boot/dts/Makefile
+++ b/arch/arm/boot/dts/Makefile
@@ -413,13 +413,11 @@ dtb-$(CONFIG_SOC_IMX6Q) += \
 	imx6dl-sabreauto-enetirq.dtb \
 	imx6dl-sabreauto-flexcan1.dtb \
 	imx6dl-sabreauto-gpmi-weim.dtb \
-	imx6dl-sabreauto-optee.dtb \
 	imx6dl-sabrelite.dtb \
 	imx6dl-sabresd.dtb \
 	imx6dl-sabresd-btwifi.dtb \
 	imx6dl-sabresd-hdcp.dtb \
 	imx6dl-sabresd-ldo.dtb \
-	imx6dl-sabresd-optee.dtb \
 	imx6dl-savageboard.dtb \
 	imx6dl-ts4900.dtb \
 	imx6dl-ts7970.dtb \
@@ -512,14 +510,12 @@ dtb-$(CONFIG_SOC_IMX6Q) += \
 	imx6q-sabreauto-enetirq.dtb \
 	imx6q-sabreauto-flexcan1.dtb \
 	imx6q-sabreauto-gpmi-weim.dtb \
-	imx6q-sabreauto-optee.dtb \
 	imx6q-sabrelite.dtb \
 	imx6q-sabresd.dtb \
 	imx6q-sabresd-btwifi.dtb \
 	imx6q-sabresd-hdcp.dtb \
 	imx6q-sabresd-ldo.dtb \
 	imx6q-sabresd-enetirq.dtb \
-	imx6q-sabresd-optee.dtb \
 	imx6q-savageboard.dtb \
 	imx6q-sbc6x.dtb \
 	imx6q-tbs2910.dtb \
@@ -548,13 +544,11 @@ dtb-$(CONFIG_SOC_IMX6Q) += \
 	imx6qp-sabreauto-ecspi.dtb \
 	imx6qp-sabreauto-flexcan1.dtb \
 	imx6qp-sabreauto-gpmi-weim.dtb \
-	imx6qp-sabreauto-optee.dtb \
 	imx6qp-sabresd.dtb \
 	imx6qp-sabresd-btwifi.dtb \
 	imx6qp-sabresd-hdcp.dtb \
 	imx6qp-sabresd-ldo.dtb \
 	imx6qp-sabresd-ldo-pcie-cert.dtb \
-	imx6qp-sabresd-optee.dtb \
 	imx6qp-tx6qp-8037.dtb \
 	imx6qp-tx6qp-8037-mb7.dtb \
 	imx6qp-tx6qp-8137.dtb \
@@ -567,18 +561,15 @@ dtb-$(CONFIG_SOC_IMX6SL) += \
 	imx6sl-evk-ldo.dtb \
 	imx6sl-evk-csi.dtb \
 	imx6sl-evk-uart.dtb \
-	imx6sl-evk-optee.dtb \
 	imx6sl-warp.dtb
 dtb-$(CONFIG_SOC_IMX6SX) += \
 	imx6sx-nitrogen6sx.dtb \
 	imx6sx-14x14-arm2.dtb \
 	imx6sx-sabreauto.dtb \
 	imx6sx-sabreauto-m4.dtb \
-	imx6sx-sabreauto-optee.dtb \
 	imx6sx-sdb-reva.dtb \
 	imx6sx-sdb-reva-ldo.dtb \
 	imx6sx-sdb-sai.dtb \
-	imx6sx-sdb-optee.dtb \
 	imx6sx-sdb.dtb \
 	imx6sx-sdb-btwifi.dtb \
 	imx6sx-sdb-emmc.dtb \
@@ -610,7 +601,6 @@ dtb-$(CONFIG_SOC_IMX6UL) += \
 	imx6ul-14x14-evk-emmc.dtb \
 	imx6ul-14x14-evk-gpmi-weim.dtb \
 	imx6ul-14x14-evk-usb-certi.dtb \
-	imx6ul-14x14-evk-optee.dtb \
 	imx6ul-geam.dtb \
 	imx6ul-isiot-emmc.dtb \
 	imx6ul-isiot-nand.dtb \
@@ -631,7 +621,6 @@ dtb-$(CONFIG_SOC_IMX6UL) += \
 	imx6ul-14x14-evk-pf1550.dtb \
 	imx6ul-9x9-evk.dtb \
 	imx6ul-9x9-evk-csi.dtb \
-	imx6ul-9x9-evk-optee.dtb \
 	imx6ul-9x9-evk-btwifi.dtb \
 	imx6ul-9x9-evk-btwifi-oob.dtb \
 	imx6ul-9x9-evk-ldo.dtb
@@ -660,7 +649,6 @@ dtb-$(CONFIG_SOC_IMX6ULL) += \
 	imx6ull-14x14-evk-emmc.dtb \
 	imx6ull-14x14-evk-gpmi-weim.dtb \
 	imx6ull-14x14-evk-usb-certi.dtb \
-	imx6ull-14x14-evk-optee.dtb \
 	imx6ull-9x9-evk.dtb \
 	imx6ull-9x9-evk-btwifi.dtb \
 	imx6ull-9x9-evk-btwifi-oob.dtb \
@@ -673,7 +661,6 @@ dtb-$(CONFIG_SOC_IMX6SLL) += \
 	imx6sll-lpddr3-arm2-spdif.dtb \
 	imx6sll-evk.dtb \
 	imx6sll-evk-reva.dtb \
-	imx6sll-evk-optee.dtb \
 	imx6sll-evk-btwifi.dtb
 dtb-$(CONFIG_SOC_IMX7D) += \
 	imx7d-cl-som-imx7.dtb \
@@ -689,7 +676,6 @@ dtb-$(CONFIG_SOC_IMX7D) += \
 	imx7d-sdb-sht11.dtb \
 	imx7d-sdb-qspi.dtb \
 	imx7d-sdb-mipi-dsi.dtb \
-	imx7d-sdb-optee.dtb \
 	imx7d-sdb-reva.dtb \
 	imx7d-sdb-reva-epdc.dtb \
 	imx7d-sdb-reva-gpmi-weim.dtb \
diff --git a/arch/arm/boot/dts/imx6dl-sabreauto-optee.dts b/arch/arm/boot/dts/imx6dl-sabreauto-optee.dts
deleted file mode 100644
index 2362784..0000000
--- a/arch/arm/boot/dts/imx6dl-sabreauto-optee.dts
+++ /dev/null
@@ -1,13 +0,0 @@
-/*
- * Copyright 2017 NXP
- *
- * This program is free software; you can redistribute it and/or modify
- * it under the terms of the GNU General Public License version 2 as
- * published by the Free Software Foundation.
- */
-
-#include "imx6dl-sabreauto.dts"
-
-&ocram {
-	reg = <0x00905000 0x13000>;
-};
diff --git a/arch/arm/boot/dts/imx6dl-sabresd-optee.dts b/arch/arm/boot/dts/imx6dl-sabresd-optee.dts
deleted file mode 100644
index 7b5af9a..0000000
--- a/arch/arm/boot/dts/imx6dl-sabresd-optee.dts
+++ /dev/null
@@ -1,13 +0,0 @@
-/*
- * Copyright 2017 NXP
- *
- * This program is free software; you can redistribute it and/or modify
- * it under the terms of the GNU General Public License version 2 as
- * published by the Free Software Foundation.
- */
-
-#include "imx6dl-sabresd.dts"
-
-&ocram {
-	reg = <0x00905000 0x13000>;
-};
diff --git a/arch/arm/boot/dts/imx6q-sabreauto-optee.dts b/arch/arm/boot/dts/imx6q-sabreauto-optee.dts
deleted file mode 100644
index 6b684bd..0000000
--- a/arch/arm/boot/dts/imx6q-sabreauto-optee.dts
+++ /dev/null
@@ -1,14 +0,0 @@
-/*
- * Copyright 2017 NXP
- *
- * This program is free software; you can redistribute it and/or modify
- * it under the terms of the GNU General Public License version 2 as
- * published by the Free Software Foundation.
- */
-
-#include "imx6q-sabreauto.dts"
-
-&ocram {
-	/* Reserve the high 32K for OP-TEE */
-	reg = <0x00905000 0x33000>;
-};
diff --git a/arch/arm/boot/dts/imx6q-sabresd-optee.dts b/arch/arm/boot/dts/imx6q-sabresd-optee.dts
deleted file mode 100644
index 1fa1094..0000000
--- a/arch/arm/boot/dts/imx6q-sabresd-optee.dts
+++ /dev/null
@@ -1,14 +0,0 @@
-/*
- * Copyright 2017 NXP
- *
- * This program is free software; you can redistribute it and/or modify
- * it under the terms of the GNU General Public License version 2 as
- * published by the Free Software Foundation.
- */
-
-#include "imx6q-sabresd.dts"
-
-&ocram {
-	/* Reserve the high 32K for OP-TEE */
-	reg = <0x00905000 0x33000>;
-};
diff --git a/arch/arm/boot/dts/imx6qp-sabreauto-optee.dts b/arch/arm/boot/dts/imx6qp-sabreauto-optee.dts
deleted file mode 100644
index 1d9cdcd..0000000
--- a/arch/arm/boot/dts/imx6qp-sabreauto-optee.dts
+++ /dev/null
@@ -1,14 +0,0 @@
-/*
- * Copyright 2017 NXP
- *
- * This program is free software; you can redistribute it and/or modify
- * it under the terms of the GNU General Public License version 2 as
- * published by the Free Software Foundation.
- */
-
-#include "imx6qp-sabreauto.dts"
-
-/* TODO: This needs to be changed to use OCRAM3 */
-&ocram {
-	reg = <0x00905000 0x33000>;
-};
diff --git a/arch/arm/boot/dts/imx6qp-sabresd-optee.dts b/arch/arm/boot/dts/imx6qp-sabresd-optee.dts
deleted file mode 100644
index 8a036e9..0000000
--- a/arch/arm/boot/dts/imx6qp-sabresd-optee.dts
+++ /dev/null
@@ -1,14 +0,0 @@
-/*
- * Copyright 2017 NXP
- *
- * This program is free software; you can redistribute it and/or modify
- * it under the terms of the GNU General Public License version 2 as
- * published by the Free Software Foundation.
- */
-
-#include "imx6qp-sabresd.dts"
-
-/* TODO: This needs to be changed to use OCRAM3 */
-&ocram {
-	reg = <0x00905000 0x33000>;
-};
diff --git a/arch/arm/boot/dts/imx6sl-evk-optee.dts b/arch/arm/boot/dts/imx6sl-evk-optee.dts
deleted file mode 100644
index 10a57b8..0000000
--- a/arch/arm/boot/dts/imx6sl-evk-optee.dts
+++ /dev/null
@@ -1,13 +0,0 @@
-/*
- * Copyright 2017 NXP.
- *
- * This program is free software; you can redistribute it and/or modify
- * it under the terms of the GNU General Public License version 2 as
- * published by the Free Software Foundation.
- */
-
-#include "imx6sl-evk.dts"
-
-&ocram {
-	reg = <0x00905000 0x13000>;
-};
diff --git a/arch/arm/boot/dts/imx6sll-evk-optee.dts b/arch/arm/boot/dts/imx6sll-evk-optee.dts
deleted file mode 100644
index c71a89b..0000000
--- a/arch/arm/boot/dts/imx6sll-evk-optee.dts
+++ /dev/null
@@ -1,13 +0,0 @@
-/*
- * Copyright 2017 NXP.
- *
- * This program is free software; you can redistribute it and/or modify
- * it under the terms of the GNU General Public License version 2 as
- * published by the Free Software Foundation.
- */
-
-#include "imx6sll-evk.dts"
-
-&ocram {
-	reg = <0x00905000 0x13000>;
-};
diff --git a/arch/arm/boot/dts/imx6sx-sabreauto-optee.dts b/arch/arm/boot/dts/imx6sx-sabreauto-optee.dts
deleted file mode 100644
index a84a09e..0000000
--- a/arch/arm/boot/dts/imx6sx-sabreauto-optee.dts
+++ /dev/null
@@ -1,23 +0,0 @@
-/*
- * Copyright 2017 NXP
- *
- * This program is free software; you can redistribute it and/or modify
- * it under the terms of the GNU General Public License version 2 as
- * published by the Free Software Foundation.
- */
-
-#include "imx6sx-sabreauto.dts"
-
-&ocram {
-	reg = <0x00905000 0x1b000>;
-};
-
-&ocrams_ddr {
-	reg = <0x00904000 0x1000>;
-};
-
-&ocrams {
-	/* 0x8f8000 are used by optee */
-	reg = <0x00900000 0x4000>;
-	clocks = <&clks IMX6SX_CLK_OCRAM>;
-};
diff --git a/arch/arm/boot/dts/imx6sx-sdb-optee.dts b/arch/arm/boot/dts/imx6sx-sdb-optee.dts
deleted file mode 100644
index 8c247cf..0000000
--- a/arch/arm/boot/dts/imx6sx-sdb-optee.dts
+++ /dev/null
@@ -1,23 +0,0 @@
-/*
- * Copyright 2017 NXP
- *
- * This program is free software; you can redistribute it and/or modify
- * it under the terms of the GNU General Public License version 2 as
- * published by the Free Software Foundation.
- */
-
-#include "imx6sx-sdb.dts"
-
-&ocram {
-	reg = <0x00905000 0x1b000>;
-};
-
-&ocrams_ddr {
-	reg = <0x00904000 0x1000>;
-};
-
-&ocrams {
-	/* 0x8f8000 are used by optee */
-	reg = <0x00900000 0x4000>;
-	clocks = <&clks IMX6SX_CLK_OCRAM>;
-};
diff --git a/arch/arm/boot/dts/imx6ul-14x14-evk-optee.dts b/arch/arm/boot/dts/imx6ul-14x14-evk-optee.dts
deleted file mode 100644
index a764769..0000000
--- a/arch/arm/boot/dts/imx6ul-14x14-evk-optee.dts
+++ /dev/null
@@ -1,14 +0,0 @@
-
-/*
- * Copyright 2017 NXP
- *
- * This program is free software; you can redistribute it and/or modify
- * it under the terms of the GNU General Public License version 2 as
- * published by the Free Software Foundation.
- */
-
-#include "imx6ul-14x14-evk.dts"
-
-&ocram {
-	reg = <0x00905000 0x13000>;
-};
diff --git a/arch/arm/boot/dts/imx6ul-9x9-evk-optee.dts b/arch/arm/boot/dts/imx6ul-9x9-evk-optee.dts
deleted file mode 100644
index cfde6b1..0000000
--- a/arch/arm/boot/dts/imx6ul-9x9-evk-optee.dts
+++ /dev/null
@@ -1,13 +0,0 @@
-/*
- * Copyright 2017 NXP
- *
- * This program is free software; you can redistribute it and/or modify
- * it under the terms of the GNU General Public License version 2 as
- * published by the Free Software Foundation.
- */
-
-#include "imx6ul-9x9-evk.dts"
-
-&ocram {
-	reg = <0x00905000 0x13000>;
-};
diff --git a/arch/arm/boot/dts/imx6ull-14x14-evk-optee.dts b/arch/arm/boot/dts/imx6ull-14x14-evk-optee.dts
deleted file mode 100644
index 4027c54..0000000
--- a/arch/arm/boot/dts/imx6ull-14x14-evk-optee.dts
+++ /dev/null
@@ -1,13 +0,0 @@
-/*
- * Copyright 2017 NXP
- *
- * This program is free software; you can redistribute it and/or modify
- * it under the terms of the GNU General Public License version 2 as
- * published by the Free Software Foundation.
- */
-
-#include "imx6ull-14x14-evk.dts"
-
-&ocram {
-	reg = <0x00905000 0x13000>;
-};
diff --git a/arch/arm/boot/dts/imx7d-sdb-optee.dts b/arch/arm/boot/dts/imx7d-sdb-optee.dts
deleted file mode 100644
index 9302a2c..0000000
--- a/arch/arm/boot/dts/imx7d-sdb-optee.dts
+++ /dev/null
@@ -1,22 +0,0 @@
-/*
- * Copyright 2017 NXP
- *
- * This program is free software; you can redistribute it and/or modify
- * it under the terms of the GNU General Public License version 2 as
- * published by the Free Software Foundation.
- */
-#include "imx7d-sdb.dts"
-
-&ocram {
-	reg = <0x00905000 0x1b000>;
-};
-
-&ocrams_ddr {
-	reg = <0x00904000 0x1000>;
-};
-
-&ocrams {
-	/* 0x180000 are used by optee */
-	reg = <0x00900000 0x4000>;
-	clocks = <&clks IMX7D_OCRAM_CLK>;
-};
-- 
1.7.9.5

